app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+from typing import List
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
```

Comment:
It would be better to use .db3 file extension which is associated with SQLite version 3